collector/qdisc_linux.go

```diff
@@ -84,7 +92,8 @@ func NewQdiscStatCollector(logger log.Logger) (Collector, error) {
 			"Number of bytes currently in queue to be sent.",
 			[]string{"device", "kind"}, nil,
 		), prometheus.GaugeValue},
-		logger: logger,
+		logger:       logger,
+		deviceFilter: newNetDevFilter(*collectorQdiskDeviceExclude, *collectorQdiskDeviceExclude),
```

Comment:
This needs to be rebased in order to pick up the new generic `newDeviceFilter()` function. `newNetDevFilter()` doesn't exist anymore